code/src/Components/Options.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+const Options = ({ setOptions, options }) => {
+  return (
+    <div className="survey-item options">
+      <label className="section-heading" htmlFor="option">
+        Choose city for the event
+      </label>
+      <select
+        className="optionCity"
+        onChange={(e) => {
+          setOptions(e.target.value);
+        }}
+        id="option"
+        value={options}
+      >
+        <option disabled>Choose below</option>
```

Comment:
I noticed that even though you set `disabled` here you can still choose "choose below" as an option. I found this that maybe can work instead: `disabled={true}`.